addons/mrp_repair/views/production_views.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="mrp_production_form_view_inherit" model="ir.ui.view">
+            <field name="name">mrp.production.form.inherit</field>
+            <field name="model">mrp.production</field>
+            <field name="inherit_id" ref="mrp.mrp_production_form_view"/>
+            <field name="arch" type="xml">
+                <xpath expr="//div[@name='button_box']" position="inside">
+                    <button
+                        name="action_repair_orders"
+                        icon="fa-wrench"
+                        type="object"
+                        invisible="mto_repairs_count == 0"
+                        groups="stock.group_stock_user"
+                        class="oe_stat_button"
+                        help="Repair">
```

Comment:
does this "help" attribute on the buttons do anything?

Reply:
![image](https://github.com/user-attachments/assets/c099697e-721f-4542-8d51-9e9bbc86e977)
is it useful? It's display the same name that are inside the button